README.rst

```diff
@@ -1,9 +1,10 @@
+:warning: *dwavebinarycsp* is deprecated. For solving problems with constraints,
+    we recommend using the hybrid solvers in the Leap service. You can find
```

Comment:
```suggestion
    we recommend using the hybrid solvers in the Leap&trade; service. You can find
```

First use for the code

Reply:
Will that work in GH flavoured markdown?

Reply:
https://stackoverflow.com/questions/12753022/how-do-you-escape-the-trademark-or-copyright-symbol-in-markdown